doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
Oh, I think I understand now. Basically you're just saying take out the leading ">>>" and "..." from the snippets presented to the user in order to reduce clutter? Because we still desire to show how to call `configureLoggers`, right?

That would work for the `configureLoggers` call, but it seems like the "interactive example" format is actually about the clearest way to present the results of configuring the loggers---that is, how else would you demonstrate to the reader that 'logger.info(...)' produces output on stdout while 'logger.debug(...)' does not?